moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -90,39 +90,39 @@ void RobotModel::buildModel(const urdf::ModelInterface& urdf_model, const srdf::
   link_geometry_count_ = 0;
   variable_count_ = 0;
   model_name_ = urdf_model.getName();
-  ROS_INFO_NAMED(LOGNAME, "Loading robot model '%s'...", model_name_.c_str());
+  ROS_INFO_NAMED(LOGNAME.c_str(), "Loading robot model '%s'...", model_name_.c_str());
```

Comment:
i think instead we should change the type of LOGNAME to char to make all these lines more succinct

Reply:
Here as well, I believe this should be addressed in the ROS2 logging library interfaces.
Working around every single inconvenient interface in ROS2 is not a good idea for porting the entire MoveIt project.

Reply:
In an email exchange with acutronic i encouraged they make atomic changes like this in one large PR to all of the moveit codebase, using find-replace commands. So porting will be faster and not manual.

Reply:
See comment above